src/lib/docs/AccountKit/Permission.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Permission
+description: A Permission instance is a wrapper for an Antelope blockchain account permission level.
+category: AccountKit
+published: true
+order: 12
+---
+
+# Permission
+
+Each `Permission` instance represents a permission level on an Antelope blockchain account. It offers a convenient way to read the permission level's details, as well as modify it.
```

Comment:
"represents a named permission level associated to an Antelope blockchain account."

We have `PermissionLevel` already and stating these are "permission levels" is kinda confusing.

Reply:
Yeah, you're right that it's a bit confusing 😅 I'll stick to using `permission` 👍